renv.lock

```diff
@@ -789,8 +829,8 @@
       "RemoteHost": "api.github.com",
       "RemoteUsername": "ccao-data",
       "RemoteRepo": "lightsnip",
-      "RemoteRef": "master",
-      "RemoteSha": "f877e887a51f8d423d5ca80719e345e31c4c311c",
+      "RemoteRef": "jeancochrane/avoid-loading-or-saving-record-evals-if-not-present",
```

Comment:
Note to self: Switch this back to `master` after https://github.com/ccao-data/lightsnip/pull/13 lands.

Reply:
Done in e6a69a956990ab169fa295a31f42387e6d051623.